frontend/build.gradle

```diff
@@ -16,12 +16,17 @@ repositories {
 dependencies {
 
     testCompile group: 'junit', name: 'junit', version: '4.12'
-    implementation 'org.springframework.boot:spring-boot-starter-web'
+    //implementation 'org.springframework.boot:spring-boot-starter-web'
     compile project(':domain')
     testImplementation('org.springframework.boot:spring-boot-starter-test') {
         exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
         implementation 'com.google.code.gson:gson:2.8.6'
-
+        compile 'org.springframework.boot:spring-boot-starter-web:2.0.1.RELEASE'
+//        compile 'org.springframework.boot:spring-boot-starter-data-jpa:2.0.1.RELEASE'
```

Comment:
Same comment about deleting unused comments / comments that effectively do the same as deleting code